src/app/containers/NewStoryPromo/PromoOrganisms/README.md

```diff
@@ -0,0 +1 @@
+Attempt on using atomic design.
```

Comment:
They are already getting styled according to the pageType (Optimo Articles). For now, I kept everything in one Folder since it's much easier to visualise for me. But you can imagine that the Entire TopStoryPromo will be co-located with the Page.
And Promo and PromoAtoms will be located in the components folder.
I know I am making this confusing, but helps me when showing the work I am doing to have everything in one folder 😓